README.md

```diff
@@ -10,7 +10,7 @@
 
 ### Llive Demo
 
-- https://mbonabucya.github.io/Portfolio/html/index.html
+- https://mbonabucya.github.io/Portfolio/
 
```

Comment:
- [ **OPTIONAL** ] Good job with your app deployment, now everyone can see your application online.  So, I think it would be better to use this format below to make your documentation looks smart(this will not show the link, but the name in the [square brackets])
```bash
[Live Demo Link](https://mbonabucya.github.io/Portfolio/)
```